README.md

```diff
@@ -20,7 +20,7 @@ This software is EXPERIMENTAL software. USE ON MAINNET AT YOUR OWN RISK.
 
 By default this software will use your existing Bitcoin wallet, including spending bitcoins contained therein (for example for transaction fees or trading).
 The protocol and transaction processing rules for the Omni Layer are still under active development and are subject to change in future.
-Omni Core should be considered an alpha-level product, and you use it at your own risk. Neither the Omni Foundation nor the Omni Core developers assumes any responsibility for funds misplaced, mishandled, lost, or misallocated.
+Omni Core should be considered an beta-level product, and you use it at your own risk. Neither the Omni Foundation nor the Omni Core developers assumes any responsibility for funds misplaced, mishandled, lost, or misallocated.
```

Comment:
Heh - you have a point!  I didn't notice that...  Hmm - I really don't know which is the right term for the current state :(